apps/betterangels-backend/hmis/schema.py

```diff
@@ -50,10 +49,36 @@ def hmis_client_profile(self, info: Info, hmis_id: str) -> HmisClientProfileType
         permission_classes=[IsAuthenticated],
     )
 
+    @strawberry_django.field(permission_classes=[IsAuthenticated])
+    def hmis_note(self, info: Info, client_hmis_id: str, note_hmis_id: str) -> HmisNoteType:
+        hmis_api_bridge = HmisRestApiBridge(info=info)
+
+        note_data = hmis_api_bridge.get_note(
+            client_hmis_id=client_hmis_id,
+            note_hmis_id=note_hmis_id,
+        )
+
+        hmis_client_profile_id = HmisClientProfile.objects.get(hmis_id=client_hmis_id).pk
+
+        hmis_note, _ = HmisNote.objects.filter(
+            hmis_id=note_hmis_id,
+            hmis_client_profile_id=hmis_client_profile_id,
+            client_hmis_id=client_hmis_id,
+        ).update_or_create(
+            hmis_id=note_hmis_id,
+            hmis_client_profile_id=hmis_client_profile_id,
+            client_hmis_id=client_hmis_id,
+            defaults={**note_data},
+        )
+
+        return cast(HmisNoteType, hmis_note)
+
+    hmis_notes: OffsetPaginated[HmisNoteType] = strawberry_django.offset_paginated(permission_classes=[IsAuthenticated])
```

Comment:
This is a security issue.  I can be any type of user and retrieve hmis notes

Reply:
added this ticket https://betterangels.atlassian.net/browse/DEV-2259